### PR TITLE
Add 1.7.0-alpha01 to changelog

### PR DESCRIPTION
**Pull request by MatkovIvan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,95 @@
...
+
+### Android
+
+- [Minimal supported AGP raised to 8.1.0](https://github.com/JetBrains/compose-multiplatform/pull/4840)
+
+## Features
+
+### iOS
+
+- [Added source-compatible `accessibilityEnabled: Boolean = true` argument to `UIKitView` and `UIKitViewController`](https://github.com/JetBrains/compose-multiplatform-core/pull/1350)
```
Comment (igordmn) [inline CHANGELOG.md:22]:
```suggestion
- [Added `accessibilityEnabled: Boolean = true` argument to `UIKitView` and `UIKitViewController`](https://github.com/JetBrains/compose-multiplatform-core/pull/1350)
```

Comment (igordmn) [reply CHANGELOG.md:22]: This is our default. We should write only if it is source-incompatible, in `Breaking changes`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,95 @@
...
+- [`preferredStatusBarStyle`, `preferredStatysBarAnimation` and `prefersStatusBarHidden` are added to `ComposeUIViewControllerDelegate` to allow status bar appearance modification](https://github.com/JetBrains/compose-multiplatform-core/pull/1378)
+
+### Desktop
+
+- [Add constructor with `reducePresentationLatency` to `ComposePanel`. Setting this flag to `true` gives a hint to renderer implementation to reduce the latency between the input and visual changes in exchange for possible screen tearing](https://github.com/JetBrains/compose-multiplatform-core/pull/1377)
```
Comment (igordmn) [inline CHANGELOG.md:27]: It is the old Release Notes. Asked for actualisation in https://github.com/JetBrains/compose-multiplatform-core/pull/1377#issuecomment-2200250463

Comment (elijah-semyonov) [reply CHANGELOG.md:27]: Updated

**Review by igordmn — APPROVED** [review]

**Pull request merged**